lists/CI-MOI

```diff
@@ -0,0 +1,46 @@
+{
+  "name": {
+    "en": "Ministry of the Interior (Ivory Coast)",
+    "local": "Ministère de l'Intérieur (Côte d'Ivoire)"
+  },
+  "url": "http://http://interieur.gouv.ci/",
+  "description": {
+    "en": "The Ministry of the Interior and Security is the ministerial department of the Ivorian Government responsible for the administration of the territory and public freedoms, the organization of administrative districts, civil status and associations."
+  },
+  "coverage": [
+    "CI"
+  ],
+  "subnationalCoverage": [],
+  "structure": [],
```

Comment:
We'll need something in the 'structure' field about what kinds of organisations are registered with this agency.

Reply:
And that information would also help if added to the 'description' property.